src/vaadin-grid-sort-mixin.js

```diff
@@ -51,31 +51,66 @@ export const SortMixin = (superClass) =>
     /** @private */
     _onSorterChanged(e) {
       const sorter = e.target;
+      e.stopPropagation();
+      if (this._isUpdatingSorter) return;
+      try {
+        this._isUpdatingSorter = true;
```

Comment:
Is it possible to add a check for sorter in `__updateSorter` to avoid using flag over here?

Reply:
Done